src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -627,7 +627,7 @@ private void ReleaseWriteLock()
 
         private async ValueTask SendSettingsAckAsync()
         {
-            await AcquireWriteLockAsync().ConfigureAwait(false);
+            await AcquireWriteLockAsync(CancellationToken.None).ConfigureAwait(false);
```

Comment:
FWIW, you can also do `default`.  They're the same, since None will be inlined and return default, so whichever you prefer stylistically.  We should just be consistent throughout.

Reply:
AcquireWriteLockAsync could also be defined to have its `CancellationToken cancellationToken = default`, i.e. an optional argument, in which case the call site wouldn't need to be modified in this case.

Reply:
I've updated the change to use `default`, so this should be fixed in the next commit.